server/monitor/src/main/java/org/apache/accumulo/monitor/rest/trace/TracesResource.java

```diff
@@ -176,12 +193,15 @@ public Void run() {
   @Path("show/{id}")
   @GET
   public TraceList getTracesType(@PathParam("id") String id) throws Exception {
-    TraceList traces = new TraceList(id);
 
-    if (id == null) {
+    if (StringUtils.isEmpty(id)) {
```

Comment:
I don't know what type of ID this is referring to but if it is a table or namespace, should create ID type objects.